python/ray/serve/tests/test_schema.py

```diff
@@ -617,6 +618,11 @@ def test_remove_app_name_from_deployment_names(self):
         with pytest.raises(AssertionError):
             config.remove_app_name_from_deployment_names()
 
+    def test_serve_application_import_path_required(self):
+        # If no import path is specified, this should not parse successfully
+        with pytest.raises(ValidationError):
+            ServeApplicationSchema.parse_obj({"host": "127.0.0.1", "port": 8000})
```

Comment:
hmm, actually i think it should be valid to submit just the serve-level config but no applications. this means just start serve but don't deploy anything yet.

Reply:
Ah I see. We allow this for `ServeDeploySchema`, i.e. start serve but deploy zero applications. Do you think we need to support this for `ServeApplicationSchema` as well? 

The only use-case I can see for this is allow users who still use the `ServeApplicationSchema` format for their config files to submit a config without deploying an actual application, but we would actually need to add support for that because currently if a `ServeApplicationSchema` config is submitted without an import path, it will fail later on when running `import_attr`.

Reply:
Yes I think we need to support this.

Imagine if there is a shared cluster where multiple users are applying different applications (some users have expressed intent to do this). The first step may be to set up just the cluster and start serve, then apply with the actual applications later. I'd be surprised if this isn't supported as a user.

Reply:
If there are multiple applications involved, they should be using ServeDeploySchema, right? In which case they just have an empty list of applications

Reply:
Oh I'm sorry the names are confusing me.... yes that's right, it should be the new schema w/ an empty list.

Reply:
My understanding is `ServeApplicationSchema` controls the behavior of the _existing_ API. If that's correct, then having import_path required makes sense to me.